examples/randomized-benchmark/DRAG-optimization/configuration.py

```diff
@@ -1,40 +1,72 @@
 import numpy as np
 
-gauss_len = 100
+gauss_len = 16
 gaussian_amp = 0.2
 
 
 def gauss(amplitude, mu, sigma, delf, length):
-    t = np.linspace(-length / 2, length / 2, length)
```

Comment:
I think I like the symmetric version better, because then mu=0 means you get a normal gaussian. This is also the convention for scipy and other packages...
(I would actually remove the "mu" parameter, nobody ever changes it!)

But I'm open to discussion, convince me otherwise!

Reply:
I was following what was reported in the paper Chen in the comments. It is really not relevant for gaussian, but for cosine you would need to do it from 0 to lenght